operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -1187,4 +1252,33 @@ protected boolean hasCaCertGenerationChanged(Secret secret) {
         }
         return false;
     }
+
+    /**
+     * Generates hash stub of the certificate which is used to track when the certificate changes and rolling update needs to be triggered.
+     *
+     * @param certSecret    Secrets with the certificate
+     * @param key           Key under which the certificate is stored in the Secret
+     *
+     * @return              Hash stub of the certificate
+     */
+    public static String getCertificateThumbprint(Secret certSecret, String key) {
+        try {
+            X509Certificate cert = Ca.cert(certSecret, key);
+            return Util.hashStub(cert.getEncoded());
+        } catch (CertificateEncodingException e) {
+            throw new RuntimeException("Failed to get certificate hashStub of " + key + " from Secret " + certSecret.getMetadata().getName(), e);
+        }
+    }
+
+    /**
+     * Generates hash stub of the broker certificate which is used to track when the certificate changes and rolling update needs to be triggered.
+     *
+     * @param certSecret    Secrets with the certificate
+     * @param podName       Broker pod name
+     *
+     * @return              Hash stub of the certificate
+     */
+    public static String getCertificateThumbprintForBroker(Secret certSecret, String podName) {
```

Comment:
Same as above. Should it really be here? Given it is directly ried to brokers, then maybe if nothing else `ClusterCa` is a better place for this? But strictly speaking, it might not be that much CA related as just a utility method?

Reply:
Done